operator/bundle/manifests/seldon-operator.clusterserviceversion.yaml

```diff
@@ -435,6 +436,10 @@ spec:
                 command:
                 - /manager
                 env:
+                - name: MANAGER_LEADER_ELECTION_ID
+                  value: a33bd623.machinelearning.seldon.io
```

Comment:
is this expected to always be a33bd623

Reply:
It can be set in latest [Helm chart.](https://github.com/SeldonIO/seldon-core/blob/8eaf8fe4001a43561704c1ecdee379572048fb51/helm-charts/seldon-core-operator/values.yaml#L84)